bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/checksum/DirectMemoryCRC32Digest.java

```diff
@@ -83,6 +83,9 @@ public void update(ByteBuf buf) {
             updateBytesMethod = CRC32.class.getDeclaredMethod("updateBytes", int.class, byte[].class, int.class,
                     int.class);
             updateBytesMethod.setAccessible(true);
+        } catch (RuntimeException e) {
```

Comment:
let's do these changes in a separate PR for master and cherry-pick it into branch-4.14

Reply:
it's okay. it should be apply cleanly on 4.14 branch. All these changes are needed to make spotbugs passes